ui/server.py

```diff
@@ -176,9 +183,9 @@ def download_rule():
             'description': rule['description']
         })
 
-    with open('./backend/Downloadrules.yml', 'w') as file:
+    with open(os.path.join(APP_ROOT, './backend/Downloadrules.yml'), 'w') as file:
```

Comment:
is this correct, or should it be changed as follows?
```
with open(os.path.join(APP_ROOT, 'backend/Downloadrules.yml'), 'w') as file:
```

Reply:
From my experience, `os.path.join` accepts a path-like object on both the arguments, and takes care of the current directory notation. I tested this both on Windows and Ubuntu.
Shall I change this and remove the `./` just to be sure?

Reply:
It's fine, let's keep it